pjlib/src/pj/ssl_sock_ossl.c

```diff
@@ -1,4 +1,4 @@
-/* $Id$ */
+/* $Id$ */ 
```

Comment:
accidental modification?

Reply:
Yes, sorry about that. Have pushed an update.